/usr/lib/amd64 getting added as RUNPATH/RPATH to binaries and libraries, causing gmake publish to fail.   Do not append the system 64-bit library path to RPATH.

--- hdf5-2.1.1/config/HDFMacros.cmake.orig
+++ hdf5-2.1.1/config/HDFMacros.cmake
@@ -505,8 +505,6 @@
         "@loader_path/../${${package_prefix}_INSTALL_LIB_DIR}"
         "@loader_path/"
     )
-  else ()
-    list (APPEND CMAKE_INSTALL_RPATH "\$ORIGIN/../${${package_prefix}_INSTALL_LIB_DIR}:\$ORIGIN/")
   endif ()
 
   if (DEFINED ADDITIONAL_CMAKE_PREFIX_PATH AND EXISTS "${ADDITIONAL_CMAKE_PREFIX_PATH}")
